src/main/java/teamroots/embers/recipe/RecipeRegistry.java

```diff
@@ -260,7 +260,7 @@ public void initEarly(RegistryEvent.Register<IRecipe> event) {
 		EmbersAPI.registerCatalysisFuel(new OreIngredient("gunpowder"),3.0);
 		EmbersAPI.registerCatalysisFuel(new OreIngredient("dustGlowstone"),4.0);
 
-		EmbersAPI.registerBoilerFluid(FluidRegistry.WATER,FluidRegistry.getFluid("steam"),5.0);
+		EmbersAPI.registerBoilerFluid(FluidRegistry.WATER,FluidRegistry.getFluid("steam"),ConfigManager.miniBoilerWaterMultiplier);
```

Comment:
this should really be crafttweaker integration. If you don't want to touch that, could you remove this change for now?

Reply:
Okay, I've added crafttweaker integration for steam engine fluids and boiler fluids now, just remove this change and the associated config option and i'll merge